platform/windows/detect.py

```diff
@@ -217,9 +217,20 @@ def configure_msvc(env, manual_msvc_config):
 
     ## Libs
 
-    LIBS = ['winmm', 'opengl32', 'dsound', 'kernel32', 'ole32', 'oleaut32',
+    LIBS = ['winmm', 'dsound', 'kernel32', 'ole32', 'oleaut32',
             'user32', 'gdi32', 'IPHLPAPI', 'Shlwapi', 'wsock32', 'Ws2_32',
-	    'shell32', 'advapi32', 'dinput8', 'dxguid', 'imm32', 'bcrypt','Avrt']
+            'shell32', 'advapi32', 'dinput8', 'dxguid', 'imm32', 'bcrypt', 'Avrt']
+
+    env.Prepend(CPPPATH=['#thirdparty/vulkan/include/', "#thirdparty/vulkan/registry/"])
```

Comment:
This should likely be handled directly in `drivers/vulkan_loader/SCsub` (or `drivers/vulkan/SCsub` if moved together), I'll have a look at it.

Reply:
Fixed by #31102.